### ISSUE TITLE
Alt text and long description boxes overlap when using keyboard to select

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#93]
> **Migrated from GitLab**
> **Original Author:** Dione Mentis (`@DioneMentis`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/394

## Expected behaviour

For accessibility purposes, users should be able to tab to select and insert text into "alt text" and "long description"

## Current behaviour

Tabbing works, however the "long description" box overlaps with "alt text" box so it looks like the user is editing the wrong element.

## Steps to reproduce

1. Within editor use keyboard to select the image (:arrow_up_small: or :arrow_down_small: arrow depending where the image is)
2. tab to select "alt text"
3. tab to select "long description"

## Testing environment

This happens in Version 0.9.37 of Wax

## QA Steps 

[To be completed by Coko]

## Scheduling

[To be completed by Coko]

* Dependencies: [provide issue numbers]
* Development estimate: [provide in hours]

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Dione Mentis (`DioneMentis`)
> **Date:** 2024-07-12T10:11:06.202Z

thanks `vukilelanga`  this is fixed
